fastlane/lib/fastlane/actions/update_project_provisioning.rb

```diff
@@ -63,6 +64,11 @@ def self.run(params)
               next
             end
 
+            codesign_build_settings_keys = build_configuration.build_settings.keys.select { |key| key.to_s.match(/CODE_SIGN_IDENTITY.*/) }
```

Comment:
I think this need to be wrapped in a `if code_signing_identity` otherwise things will get cleared out if not specified 😬

Reply:
@joshdholtz Thx 😬 I have update the code